pages/glossary/included-characters.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Included characters
+key: included-characters
+---
+
+A sequence of [encoded characters](#encoded-character) is considered included in another if the latter includes any consecutive sequence of encoded characters that [matches](#matching-characters) the former.
```

Comment:
Wow, you went all in on this definition huh? Can we collapse encoded character / matches into this, so we only have one new definition? I think that's a little more manageable.

Reply:
I would like to, but can we (in the current setup, or a planned one?) link to parts of a glossary term?  Because sometimes we need one of them, sometimes the other one.

Reply:
I don't quite understand. Maybe discuss the precise capabilities with Jey on Tuesday.

Reply:
@WilcoFiers, In Alfa we have definitions like this: 

> ##Encoded character
> 
> An encoded character is...
> 
> A string of encoded characters are considered to [match]() each other if...
> 
> A string of characters are considered to be [included in]() another string if...

Where "match" and "included in" are their own definitions that you can link to if they are the ones you need specifically.

I thought it was this that you were going for, but maybe not.

Reply:
@WilcoFiers, I previously merged the two definitions, but now I split them up again, since I realised that the "matching" definition is used in other rules, and for maintenance and consistency reasons, I find it better to link out to it than duplicating it within the "included character" definition.